pom.xml

```diff
@@ -58,6 +58,11 @@
         <quarkus.version>2.14.0.Final</quarkus.version>
         <quarkus-bom.version>${quarkus.version}</quarkus-bom.version>
 
+        <!-- Workaround for https://github.com/apache/camel-quarkus/issues/4285
+             Remove once Camel Quarkus manages the same version of org.graalvm.js:js
+             as ${graalvm.version} in Quarkus -->
+        <graalvm.version>22.3.0</graalvm.version>
```

Comment:
This overrides the version from the `quarkus-bom`. We shouldn't be upgrading GraalVM version this way in the platform.

Reply:
Hm... I wonder whether I should have sent this against the `main` branch @gsmet? 
Anyway, thanks @aloubyansky for pointing out that quarkus is at 2.14.0 in main. I thought it was 2.14.1

Reply:
Hmmm, no, it's my fault, I forgot to upgrade `main` to 2.14.1. Let me add a commit to this PR.

Reply:
I removed the workaround and upgraded to Quarkus 2.14.1.Final in a separate commit.

Reply:
I also created https://github.com/quarkusio/quarkus/pull/29405 as I think we always want to have the versions consistent and it's going to be a nightmare maintaining them separately.

Reply:
> I also created https://github.com/quarkusio/quarkus/pull/29405

Let me comment there